packages/vanilla-components/src/components/text-component/text-component.tsx

```diff
@@ -2,28 +2,92 @@ import { h, Component, Prop, Element } from "@stencil/core";
 import { css } from "emotion";
 import marked from "marked";
 
+/**
+ * @uiName Text Component
+ */
 @Component({
   tag: "sqh-text-component",
   styleUrl: "text-component.scss",
 })
 export class TextComponent {
+  /**
+   * Hide the component
+   *
+   * @uiName Is Hidden
```

Comment:
Something like `Hide Button` is more appropriate, this applies to other components as well